packages/core-flows/src/product/workflows/create-products.ts

```diff
@@ -83,23 +102,6 @@ export const createProductsWorkflow = createWorkflow(
 
     createVariantPricingLinkStep(variantAndPriceSetLinks)
 
-    // TODO: Should we just refetch the products here?
-    return transform(
-      {
-        createdProducts,
-        variantAndPriceSets,
-      },
-      (data) => {
-        return data.createdProducts.map((product) => ({
-          ...product,
-          variants: product.variants?.map((variant) => ({
-            ...variant,
-            price_set: data.variantAndPriceSets.find(
-              (v) => v.variant.id === variant.id
-            )?.price_set,
-          })),
-        }))
-      }
-    )
+    return createdProducts
```

Comment:
Since we refetch after a workflow, I think it's better to not bother and reconstruct the data for now